pandas/core/computation/common.py

```diff
@@ -36,3 +36,8 @@ def _remove_spaces_column_name(name):
 
 class NameResolutionError(NameError):
     pass
+
+
+class StringMixin:
+    # TODO: delete this class. Removing this ATM caused a failure.
```

Comment:
what does it cause to fail?

Is there a reason you are using "..." instead of "pass"?

Reply:
Yeah, ``pass`` is more normal, I'll chnage that.

Reply:
If you remove the inheriance from ``StringMixin`` in ``core.computatons.scope.py::Scope`` tests will fail. Haven't been able to fix it.